### PR TITLE
Use go:embed instead of go:generate

### PR DESCRIPTION
**Pull request by Jaanki**
Signed-Off-By: Janki Chhatbar <jchhatba@redhat.com>

### CODE REVIEW

File: pkg/subctl/cmd/deploybroker.go [hunk 1]
```diff
@@ -115,6 +115,7 @@ var deployBroker = &cobra.Command{
 		status := cli.NewStatus()
 
 		status.Start("Setting up broker RBAC")
+		fmt.Printf("in deploybroker componentArr is %s\n", componentArr)
```
Comment (dfarrell07) [inline pkg/subctl/cmd/deploybroker.go:118]: This seems like it's for debugging and shouldn't stay in.

File: Makefile [hunk 2]
```diff
@@ -166,10 +166,10 @@ bin/subctl-%: generate-embeddedyamls $(shell find pkg/subctl/ -name "*.go") vend
 
 ci: generate-embeddedyamls golangci-lint markdownlint unit build images
 
-generate-embeddedyamls: generate pkg/subctl/operator/common/embeddedyamls/yamls.go
+generate-embeddedyamls: generate #pkg/subctl/operator/common/embeddedyamls/yamls.go
```
Comment (dfarrell07) [inline Makefile:169]: IMHO better to go ahead and remove it than to comment it out.

File: pkg/utils/createorupdate.go [hunk 3]
```diff
@@ -66,7 +66,8 @@ func CreateOrUpdateCRD(ctx context.Context, updater crdutils.CRDUpdater, crd *ap
 func CreateOrUpdateEmbeddedCRD(ctx context.Context, updater crdutils.CRDUpdater, crdYaml string) (bool, error) {
 	crd := &apiextensions.CustomResourceDefinition{}
 
-	if err := embeddedyamls.GetObject(crdYaml, crd); err != nil {
+	fmt.Printf("in utis.CreateOrUpdateEmbeddedCRD, crdYamls is %s crd is %s\n", crdYaml, crd)
```
Comment (dfarrell07) [inline pkg/utils/createorupdate.go:69]: More debugging to remove

File: pkg/utils/createorupdate_test.go [hunk 4]
```diff
@@ -46,7 +47,8 @@ var _ = Describe("CreateOrUpdateClusterRole", func() {
 	BeforeEach(func() {
 		clusterRole = &rbacv1.ClusterRole{}
 		// TODO skitt add our own object
-		err := embeddedyamls.GetObject(embeddedyamls.Config_rbac_submariner_globalnet_cluster_role_yaml, clusterRole)
+		//fmt.Println(embeddedyamls.Yamls)
```
Comment (dfarrell07) [inline pkg/utils/createorupdate_test.go:50]: Debugging to remove

**Comment by Jaanki** [comment]
It was supposed to be a draft :)

**Pull request closed without merging**